### ISSUE TITLE
Standalone OETH analytics

### ISSUE THREAD

**Issue by micahalcorn** [OriginProtocol/oeth.com#15]
Aside from the data that we show on the homepage, we will want an OETH version of the new analytics site that we recently launched for OUSD. The new one should live at oeth.com/analytics.

https://www.figma.com/file/BA9IMIvswYcD5xlJpcEOYG/Analytics?node-id=333-7570&t=1tlrPCf43eI1j3ok-4

**Comment by micahalcorn** [comment]
Oops, duplicate of https://github.com/OriginProtocol/ousd-analytics/issues/306 😑

**Comment by micahalcorn** [comment]
Otherwise **P1**s that are blocked by this:

- [ ] https://github.com/OriginProtocol/ousd.com/issues/146
- [ ] https://github.com/OriginProtocol/ousd.com/issues/150
- [ ] https://github.com/OriginProtocol/ousd.com/issues/151
- [ ] https://github.com/OriginProtocol/ousd.com/issues/152

**Comment by micahalcorn** [comment]
Also, https://github.com/OriginProtocol/origin-dollar/issues/1363 should probably be done before we expect to drive significant traffic to this one.

**Comment by rafaelugolini** [comment]
We need to build the OETH dune queries for this: https://dune.com/originprotocol

**Comment by smitch88** [comment]
Baseline PR out, #66 , functionally approved, awaiting design review
